src/plugins/clock/components/ClockIndicator.vue

```diff
@@ -46,16 +46,17 @@ export default {
     };
   },
   mounted() {
-    this.unlisten = ticker.listen(this.tick);
+    this.tick = raf(this.tick);
```

Comment:
In Remote-Clock mode the indicator updates the time very slowly - once every few seconds.
Is this expected?